### PR TITLE
Small fixes

### PR DESCRIPTION
**Pull request by findepi**

### CODE REVIEW

File: presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java [hunk 1]
```diff
@@ -408,7 +408,9 @@ public ColumnMetadata getColumnMetadata(ConnectorSession session, ConnectorTable
 
     private Predicate<NullableValue> convertToPredicate(TupleDomain<ColumnHandle> predicate, TpchColumn column)
     {
-        return nullableValue -> predicate.contains(TupleDomain.fromFixedValues(ImmutableMap.of(toColumnHandle(column), nullableValue)));
```
Comment (kokosing) [inline presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java]: can you please provide a test?

File: presto-main/src/main/java/com/facebook/presto/sql/planner/LiteralInterpreter.java [hunk 2]
```diff
@@ -190,6 +192,10 @@ else if (value.equals(Float.POSITIVE_INFINITY)) {
             return new BooleanLiteral(object.toString());
         }
 
+        if (type.equals(DATE)) {
```
Comment (kokosing) [inline presto-main/src/main/java/com/facebook/presto/sql/planner/LiteralInterpreter.java]: test?

File: presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java (hunk 1 above)
Comment (findepi) [reply presto-tpch/src/main/java/com/facebook/presto/tpch/TpchMetadata.java]: This is covered by `testStatsCalculatorUsesLayout` on CBO... Can you help me with this?

File: presto-main/src/main/java/com/facebook/presto/sql/planner/LiteralInterpreter.java (hunk 2 above)
Comment (findepi) [reply presto-main/src/main/java/com/facebook/presto/sql/planner/LiteralInterpreter.java]: Currently there are none direct tests of `LiteralInterpreter`. The effect of applying this change will be only visible in CBO where we do better stats math thanks to this.

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**